browser/BrowserPerf.proto

```diff
@@ -22,6 +22,7 @@ package skywalking.network.protocol.browser;
 
 option java_multiple_files = true;
 option java_package = "org.apache.skywalking.apm.network.language.agent";
+option go_package = "agent/pb/language/agent";
```

Comment:
@arugal @hanahmily What is the status of go2sky? I assume we are not using this package right now. What is the impact?

Reply:
`go2sky` does not currently refer to this repo through submodule.

Reply:
OK. But I think we should, right?

Reply:
> OK. But I think we should, right?

Agreed.

> What is the status of go2sky? I assume we are not using this package right now. What is the impact?

Now [go2sky](https://github.com/SkyAPM/go2sky/blob/aa3669fff7f880f6592eaacf5e6ca499cd74539d/reporter/grpc/common/CLR.proto#L21-L24) set `go_package` to `go2sky/reporter/grpc`, it can be modified to `agent/pb/language/agent`.

Reply:
@heyanlong What is your naming rule? They are inconsistent, And I can't see the pattern.

Reply:
> @heyanlong您的命名规则是什么？它们不一致，而且我看不到模式。

From java
network -> pb
language -> language
agent -> agent

and so
projectName/pb/language/agent

Reply:
pb is short for protobuf

Reply:
My point is you have `agent` as prefix and suffix. I think the prefix should be skywalking?